src/components/categoricaldisplay/CategoricalDisplay.jsx

```diff
@@ -1,27 +1,38 @@
 import React from 'react';
 import '../../../sass/components/_categorical-display.scss';
+import SimpleBar from 'simplebar-react';
 
-const CategoricalDisplay = ({
-    onSelectItem,
-    setScrollPanel,
-    ResultsPanel,
-    FilterPanel,
-    SearchInput,
-}) => (
-    <div className="cq-categorical-display">
-        <div className="cq-lookup-filters">
-            <SearchInput />
-            <FilterPanel />
-        </div>
-        <div
-            className="cq-scroll-panel"
-            ref={setScrollPanel}
-        >
-            <ResultsPanel
-                onSelectItem={onSelectItem}
-            />
-        </div>
-    </div>
-);
+class CategoricalDisplay extends React.Component {
```

Comment:
why do you need to make this a component?

Reply:
@brucebinary i dont. just revert it.